waspc/data/Generator/templates/server/src/queries/types.ts

```diff
@@ -0,0 +1,33 @@
+{{={= =}=}}
+import {
+  {=# allEntities =}
+  {= . =},
+  {=/ allEntities =}
+} from '../entities'
+
+import {
+  {=^ allUseAuth =}
+  Query,
+  {=/ allUseAuth =}
+  {=# someUseAuth =}
+  AuthenticatedQuery,
+  {=/ someUseAuth =}
+} from '../types'
+
+{=# operations =}
+export type {= typeName =}<Output = unknown> = 
```

Comment:
Maybe call Output -> ReturnValue? Or Returns? I see that you called it "return value" below in other comments, that is why, and I also usually use that phrase.
Not a biggie, but maybe worth considering.

Reply:
Your suggestions might make more sense, but I think the current name is most inline with TS conventions:
- `ReturnType` and `SomethingType` are used for type constructors (extractors) and they already exist in the global scope
- `Returns` is a verb, and those are generally not used to name types in TypeScript
- `ReturnValue` could be ok, but `Value` is also a little weird to encounter in a type
- 
Therefore, unless you really like `ReturnValue`, I'd stick with `Output`.